packages/help-center/src/styles.scss

```diff
@@ -54,8 +54,9 @@ $head-foot-height: 50px;
 
 		&.is-minimized {
 			min-height: $head-foot-height;
-			top: calc( 100vh - 91px ); // height of header + bottom breadcrumb bar in the editor + 16px
-
+			top: unset;
+			bottom: 0;
+			transform: unset !important; // revert dragging translation when minimized
```

Comment:
I think this is causing some problems, as it now sits on the bottom, when minimized. To me `top: calc( 100vh - 91px );` is working fine, but you could instead add some px to `bottom`, or whatever you prefer :) 
![image](https://user-images.githubusercontent.com/52076348/166888252-03ffe0ca-a426-4bae-a423-cc9d4f2d19ba.png)

Reply:
I added 41px to `bottom` because `vh` doesn't change the DevTools are open. So opening DevTools hides the help-center. This is just more convenient for us.